### PR TITLE
[ADD] partner_survey

### PR DESCRIPTION
**Pull request by JavierIniesta**
This module adds a smartbutton linked with surveys and "Surveys" page to
partner form.

It also adds survey_input_lines field to partner model. This field serves to
filter partners according to their answers in surveys.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Tried on runbot and code review

:+1:

**Comment by Ehtaga** [comment]
Review :+1:

**Comment by rafaelbn** [comment]
Tested in runbot
:+1:

File: partner_survey/views/res_partner_view.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+<record id="act_partner_survey_input" model="ir.actions.act_window">
+    <field name="res_model">survey.user_input</field>
+    <field name="view_type">form</field>
+    <field name="name">Surveys</field>
+    <field name="view_mode">tree,form,calendar,graph</field>
```
Comment (Ehtaga) [inline partner_survey/views/res_partner_view.xml:9]: Calendar and graph views don't exist

Comment (pedrobaeza) [inline partner_survey/views/res_partner_view.xml:9]: You're right. I missed that one. I have fixed it via direct commit with https://github.com/OCA/survey/commit/343e10fa3491a7e0adc1ee3810667648bd2bc33b

**Comment by rafaelbn** [comment]
@JavierIniesta review https://github.com/OCA/survey/issues/13

File: partner_survey/README.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline partner_survey/README.rst:46]: https

File: partner_survey/__init__.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline partner_survey/__init__.py:1]: Please use simple headers following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: partner_survey/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': "Partner Survey",
+    'category': 'Marketing',
+    'version': '1.0.0',
```
Comment (elicoidal) [inline partner_survey/__openerp__.py:26]: Please follow the guidelines from:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
Version should be 8.0.1.0.0

**Pull request merged**